File: Accenture_task_1/python/counting-backwards.py
def counter():
    for i in range(100, 0, -1):
        current_number = 'Testing' if i % 5 == 0 and i % 3 == 0 else \
                         'Agile' if i % 5 == 0 else \
                         'Software' if i % 3 == 0 else i
        print(current_number)

counter()

File: Accenture_task_1/js/counting-backwards.js
const counter = () => {
  for (let i = 100; i > 0; i--) {
    let currentNumber = 
      i % 5 === 0 && i % 3 === 0 ? 'Testing'
      : i % 5 === 0 ? 'Agile'
      : i % 3 === 0 ? 'Software'
      : i
    console.log(currentNumber)
  }
}

counter()